TestProject.Spec/StepDefinitions/PetTestStepDefinitions.cs

```diff
@@ -0,0 +1,35 @@
+using NUnit.Framework;
+using TechTalk.SpecFlow.Assist;
+using TestProject.Models;
+using TestProject.Utils;
+
+namespace TestProject.Spec.StepDefinitions
+{
+    [Binding]
+    public class PetTestStepDefinitions
```

Comment:
it's a minor thing, but I'd recommend a different approach for naming. Steps classes should be linked not to specific tests, but rather to page / endpoint / utilirty / etc. class 
so this one would be PetStoreSteps for example, which would help to navigate through classes and extend them once we get more and more tests